pkg/labels/selector.go

```diff
@@ -800,6 +800,22 @@ func SelectorFromSet(ls Set) Selector {
 	return internalSelector(requirements)
 }
 
+// SelectorFromValidatedSet returns a Selector which will match exactly the given Set.
+// A nil and empty Sets are considered equivalent to Everything().
+// It assumes that Set is already validated and doesn't do any validation.
+func SelectorFromValidatedSet(ls Set) Selector {
+	if ls == nil {
```

Comment:
For now I'm leaving it here for consistency with "SelectorFromSet". If you feel really strongly about it, I will send another PR with it.